test/browser.test.js

```diff
@@ -61,6 +61,28 @@ describe('browser support', function() {
     });
   });
 
+  it('loads mixins', function(done) {
+    var appDir = path.resolve(__dirname, './fixtures/browser-app');
+    var options = {
+      appRootDir: appDir,
+      mixinDirs: ['./mixins']
+    };
+
+    browserifyTestApp(options, function(err, bundlePath) {
+      if (err) return done(err);
+
+      var app = executeBundledApp(bundlePath);
+
+      var modelBuilder = app.registry.modelBuilder;
+      var registry = modelBuilder.mixins.mixins;
+      expect(Object.keys(registry)).to.eql(['Timestamping']);
```

Comment:
Inside `test/fixtures/browser-app/mixins/time-stamps.json`, you are renaming the mixin to `Timestamping`, but the model configuration refers to original name `TimeStamps`. 

I am proposing to delete `test/fixtures/browser-app/mixins/time-stamps.json` and keep the script file only.